src/standardise_records.ts

```diff
@@ -64,14 +64,12 @@ export default (key: string, json: any, txtRecordFragments: any, recordsJoined:
             for (const newSplit of dataSplit) {
                 if (newSplit === "") continue
 
-                if (!newSplit.startsWith("v")) {
-                    newRecords.push({
-                        name: record.name,
-                        data: newSplit,
-                        TTL: record.TTL,
-                        type: undefined,
-                    })
-                }
+                newRecords.push({
+                    name: record.name,
+                    data: newSplit,
+                    TTL: record.TTL,
```

Comment:
TTL should be displayed before data, it looks weird with a very large field followed by the small TTL field.

![image](https://user-images.githubusercontent.com/12371363/64263642-e2f6a580-cf27-11e9-8dc2-d113a795ca66.png)